### PR TITLE
Library utf8 tags

### PR DESCRIPTION
**Pull request by gyst**
Library tagging views were broken in solr. Now they're broken in zcatalog :-).

In a follow-up PR I will:
- deprecate the zcatalog tag views
- add robot test coverage on solr-backed library

### CODE REVIEW

**Comment by gyst** [comment]
Please hold I'm hunting futher unicode bugs

**Comment by pilz** [comment]
test please

File: src/ploneintranet/library/browser/templates/library.html [hunk 1]
```diff
@@ -158,6 +159,11 @@ <h3 tal:content="child/title">page title</h3>
 
         </div>
 
+        <p tal:condition="not:view/enabled">
```
Comment (pilz) [inline src/ploneintranet/library/browser/templates/library.html:162]: @gyst When calling the library, enable is not present and so the view gives a 500. Either a tal:on-error="nothing" or have the enabled everywhere. What do you think?

Comment (gyst) [inline src/ploneintranet/library/browser/templates/library.html:162]: On 27/08/15 02:03, Alexander Pilz wrote:

> @gyst https://github.com/gyst When calling the library, enable is not
> present and so the view gives a 500. Either a tal:on-error="nothing" or
> have the enabled everywhere. What do you think?

Ah, stupid. I've set enabled=True on everything by default.

## 

```
Guido Stevens  |  +31.43.3618933  |  http://cosent.nl

s o c i a l   k n o w l e d g e   t e c h n o l o g y
```

**Pull request merged**